config/settings/__init__.py

```diff
@@ -28,3 +28,5 @@
     CSRF_COOKIE_PATH = BASE_URL
     LANGUAGE_COOKIE_PATH = BASE_URL
     SESSION_COOKIE_PATH = BASE_URL
+except NameError:
```

Comment:
I think that this `NameError` is also a **cause** of confusion!
A setting should not change behaviour when it's there  or not there.... only the value please..
maybe keep it as before?
```py
try:
    bool(BASE_URL)
except NameError
    BASE_URL = None
# now BASE_URL has a value...
if BASE_URL:
    BASE_URL = sanitize_url(BASE_URL)
    ....
```

Reply:
Ok, you are right. Maybe

```python
BASE_URL = None

include(
    optional('base.py'),
    optional('local.py')
)

...

if BASE_URL:
    ...
```

Reply:
alright, sure, think that works! 
Where is now the `base.py` btw?

Reply:
Should I remove the one line?